### PR TITLE
Add recipe for zeus-mcmc

### PR DESCRIPTION
**Pull request by mattpitkin**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mattpitkin** [comment]
@minaskar I've created a recipe here, so that zeus can be installed via conda from the conda-forge repository (e.g., with `conda install -c conda-forge zeus-mcmc`). Are you happy for me to do this? If so would you like me to list you as a maintainer of the package?

**Comment by mattpitkin** [comment]
@conda-forge/help-python this is now ready for review.

**Comment by minaskar** [comment]
Hi @mattpitkin,

Great work Matt, thanks! Yes please add me as a maintainer.

**Comment by mattpitkin** [comment]
@minaskar No problem. I've now added you. If there's anyone else you think should be listed please let me know.

File: recipes/zeus-mcmc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b291e734ef65c05b2fb6c3f8867066133075f7a8e95c9489d21ae06062365693
+  patches:
+    # requirements and README.md aren't packaged
```
Comment (chrisburr) [inline recipes/zeus-mcmc/meta.yaml:12]: Can you open a PR or issue upstream so that this gets fixed for the next release if it hasn’t been already?

Comment (mattpitkin) [reply recipes/zeus-mcmc/meta.yaml:12]: Yep, this has been fixed in the zeus repo now https://github.com/minaskar/zeus/pull/16, so the patch won't be necessary for the next release.

**Pull request merged**